src/dynamic_graph/sot/dynamics_pinocchio/humanoid_robot.py

```diff
@@ -325,6 +325,8 @@ def traceDefaultSignals (self):
 
 
     def __init__(self, name, tracer = None):
+        self._initialize()
```

Comment:
This may cause Python errors in user scripts. I think it won't as this code was copy-pasted from somewhere else. But who knows...